apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2022-04-15T17:21:57Z"
-  build_hash: 50c64871bcaf88b9ee200eb8d6b8245fa8f675eb
-  go_version: go1.17.5
-  version: v0.18.4
-api_directory_checksum: 6972849b7a809751f78bbafa31eeb725ed5bc126
+  build_date: "2022-10-26T20:52:18Z"
+  build_hash: 5ee0ac052c54f008dff50f6f5ebb73f2cf3a0bd7
+  go_version: go1.19
+  version: 5ee0ac0
```

Comment:
We need to keep the `ack-generate` and `runtime` versions equal (make sure to fetch the code-generator tags `git fetch --tags --all` before the generation)

Reply:
Ideally we should generate the controller using version `v0.20.1` after bumping the runtime version to `v0.20.1` (PR #30)

Reply:
[](url)> We need to keep the `ack-generate` and `runtime` versions equal (make sure to fetch the code-generator tags `git fetch --tags --all` before the generation)

I've tried the command and regenerated, but the version still shows 5ee0ac0 in the file.

I guess this is the reason. Code-generator shows 5ee0ac0 as version on github now.
<img width="1726" alt="Screen Shot 2022-11-02 at 1 23 06 PM" src="https://user-images.githubusercontent.com/58649544/199595150-b6d0d65d-4056-4dd1-a32a-d11bf7838901.png">